fhir-persistence-schema/src/main/resources/db2/add_any_resource.sql

```diff
@@ -13,15 +13,17 @@
 -- p_last_updated the last_updated time given by the FHIR server
 -- p_is_deleted: the soft delete flag
 -- p_version_id: the version id if this is a replicated message
+-- o_logical_resource_id: output field returning the newly assigned logical_resource_id value
 -- o_resource_id: output field returning the newly assigned resource_id value
 -- ----------------------------------------------------------------------------
-    ( IN p_resource_type                 VARCHAR( 36 OCTETS),
-      IN p_logical_id                    VARCHAR(255 OCTETS), 
-      IN p_payload                          BLOB(1048576),
-      IN p_last_updated                TIMESTAMP,
-      IN p_is_deleted                       CHAR(  1),
-      IN p_version                           INT,
-      OUT o_logical_resource_id            BIGINT
+    ( IN p_resource_type                VARCHAR( 36 OCTETS),
+      IN p_logical_id                   VARCHAR(255 OCTETS), 
+      IN p_payload                         BLOB(1048576),
+      IN p_last_updated               TIMESTAMP,
+      IN p_is_deleted                      CHAR(  1),
+      IN p_version                          INT,
+      OUT o_logical_resource_id          BIGINT,
```

Comment:
I'm still confused about how this can be a BIGINT.  I guess its *not* the same as the `Resource.id` field of the resource?

Reply:
ok, I looked this up.  its not the first time I've been tripped up on this:
```
X_LOGICAL_RESOURCES.LOGICAL_RESOURCE_ID = the unique key of the X_LOGICAL_RESOURCES table
X_LOGICAL_RESOURCES.LOGICAL_ID = the value of the Resource.id field for this resource
```

so o_logical_resource_id corresponds to `X_LOGICAL_RESOURCES.LOGICAL_RESOURCE_ID`